src/include/parser/expression/abstract_expression.h

```diff
@@ -76,6 +84,15 @@ class AliasType {
    */
   size_t GetSerialNo() const { return serial_no_; }
 
+  /**
+   * Set the serial number of this alias
+   * @param serial_no serial number to set alias to
+   */
+  void SetSerialNo(size_t serial_no) {
```

Comment:
We should STRONG_TYPEDEF something for this. We don't like using plain number types for unique identifiers since it makes it easy to mix them up. Maybe an alias_oid_t?

Reply:
Done